docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,128 @@
+
+![TypeSymbol properties](media/type-symbol-properties.png)
+
+Try **View Converted TypeSymbol (if any)** for the same **AddExpression** node. The property grid updates indicating that although the type of the expression is `Int32`, the converted type of the expression is `Double` as shown in the following figure. This node includes converted type symbol information because the `Int32` expression occurs in a context where it must be converted to a `Double`. This conversion satisfies the `Double` type specified for the variable `x` on the left-hand side of the assignment operator.
+
+![Converted TypeSymbol properties](media/converted-type-symbol-properties.png)
+
+Finally, try **View Constant Value (if any)** for the same **AddExpression** node. The property grid shows that the value of the expression is a compile time constant with value `2`.
+
+![A constant value](media/constant-value.png)
+
+The preceding example can also be replicated in VB. Type `Dim x As Double = 1 + 1` in a VB file. Select the expression `1 + 1` in the code editor window. The visualizer highlights the corresponding **AddExpression** node in the visualizer. Repeat the preceding steps for this **AddExpression** and you should see identical results.
+
+Examine more code in VB. Update your main VB file with the following code:
+
+```vb
+Imports C = System.Console
+
+Module Program
+    Sub Main(args As String())
+        C.WriteLine()
+    End Sub
+End Module
+```
+
+This code introduces an alias named `C` that maps to the type `System.Console` at the top of the file and uses this alias inside `Main()`. Select the use of this alias, the `C` in `C.WriteLine()`, inside the method. The visualizer selects the corresponding **IdentifierName** node in the visualizer. Right-click this node and click on **View Symbol (if any)**. The property grid indicates that this identifier is bound to the type `System.Console` as shown in the following figure:
+
+![Symbol properties](media/symbol-visual-basic.png)
+
+Try **View AliasSymbol (if any)** for the same **IdentifierName** node. The property grid indicates the identifier is an alias with name `C` that is bound to the `System.Console` target. In other words, the property grid provides information regarding the **AliasSymbol** corresponding to the identifier `C`.
+
+![AliasSymbol properties](media/alias-symbol.png)
+
+Inspect the symbol corresponding to any declared type, method, property, and more. Select the corresponding node in the visualizer and click on **View Symbol (if any)**. Select the method `Sub Main()`, including the body of the method. Click on **View Symbol (if any)** for the corresponding **SubBlock** node in the visualizer. The property grid shows the **MethodSymbol** for this **SubBlock** has name `Main` with return type `Void`.
```

Comment:
"and more" seems out of -place. Just a period after property?
Since void isn't a VB term, just in case, explain that `Void` means no return value